docs/make_work.jl

```diff
@@ -0,0 +1,233 @@
+using Documenter, Oscar, DocumenterMarkdown
+
+import Documenter:
+    Anchors,
+    DocTests,
+    Documents,
+    Documenter,
+    Utilities
+
+import Documenter.Utilities: Selectors
+
+using Documenter.Builder
+import Documenter.Builder: SetupBuildDirectory, walk_navpages
+import Documenter.Documents: Document, Page
+
+function Selectors.runner(::Type{SetupBuildDirectory}, doc::Documents.Document)
```

Comment:
This function is taken from Documenter, I gather? It would be nice to add a comment here indicating where it is from (say as a link to the precise version of the file this was taken from), plus a brief hint what was changed. That way, if we want to update to a newer Documenter.jl, we have a better chance to managing it.

(And in parallel, perhaps it'd be possible to get some or all of the changes integrated into Documenter... I'd think about this if I knew what the changes are... ;-)